### ISSUE TITLE
/usr/local/bin/nelson: line 1: use strict: command not found

### ISSUE THREAD

**Issue by witwit** [SemkoDev/nelson.cli#3]
Cannot run nelson on Ubuntu 16.04 after following global installation steps in README
node -v: 9.1.0

```
nelson --neighbors nelson1.carriota.com/14600 nelson2.carriota.com/14600
/usr/local/bin/nelson: line 1: use strict: command not found
/usr/local/bin/nelson: line 3: syntax error near unexpected token `('
/usr/local/bin/nelson: line 3: `var ini = require('ini');'

```

**Comment by witwit** [comment]
Probably related to: https://stackoverflow.com/questions/34353512/node-npm-package-throw-use-strict-command-not-found-after-publish-and-install-g

pkg.js packing wrong?

**Comment by romansemko** [comment]
Fix published in v0.0.2
